### DISCUSSION TITLE
Web Open Font Format

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#146] [Q&A] [answered]
Is this format still in use? Seven year ago a WOFF Junicode was created: https://github.com/Fromager/junicode. Would it be easy and reasonable to create an up-to-date WOFF format? Personally I don't need it, but I would like to know whether to mention WOFF in a paper I'm about to write.

**Comment by psb1558** [comment]
Absolutely, only woff2 is the current format. In a Debian-based system, install the package `woff2` and run the program `woff2_compress` on any font file, or in a directory with a number of font files:
```
for f in *.ttf; do woff2_compress $f; done
```

**Reply by jsbien** [reply]
Thanks for the answer. There is also the question of subsetting. I found https://github.com/zachleat/glyphhanger which looks interesting.

**Reply by jsbien** [reply]
I forgot you write about in the manual...

**Comment by psb1558** [comment, marked as answer]
There are several web pages that will subset the font for you, and (to the extent I have tested them) they seem to work pretty well. The reason I suggest `pyftsubset` is that it gives you very fine control over the font's content. But it's a command-line program (some people can't cope with that), and it has a large number of options to sort through, so there's a learning curve.
